projects/project-player-library/src/lib/pages/attachment-listing-page/attachment-listing-page.component.ts

```diff
@@ -36,8 +36,8 @@ export class AttachmentListingPageComponent {
     let result = { project: false, task: false };
     if(attachmentType === 'image'){
       if ( this.attachments.project || this.attachments.tasks ) {
-          result.project = this.attachments.project.attachments && this.attachments.project.attachments.some((attachment: any) => attachment.type.includes(attachmentType)) || this.attachments.project.remarks;
-          result.task = this.attachments.tasks.some((task: any) => task.attachments && task.attachments.some((attachment: any) => attachment.type.includes(attachmentType))) || this.attachments.tasks.some((task:any)=>task.remarks);
+        result.project = !!(this.attachments.project.attachments && this.attachments.project.attachments.some((attachment: any) => attachment.type.includes(attachmentType)) || this.attachments.project.remarks);
```

Comment:
What is this condition ? (!!)

Reply:
!! converts this final truthy or falsy value to true or false.